platform/jobs/edxPlatformUnitCoverage.groovy

```diff
@@ -153,7 +192,7 @@ jobConfigs.each { jobConfig ->
         }
         /* Copy Artifacts from test subset jobs with build number UNIT_BUILD_NUM */
         steps {
-            for (buildNum = 1; buildNum < 7; buildNum += 1) {
+            for (buildNum = 1; buildNum < 15; buildNum += 1) {
```

Comment:
Should be `buildNum < 16` since we're starting the loop at `buildNum = 1` and there's 15 shards.

Reply:
thanks